pysqldb3/README.md

```diff
@@ -1 +1,1027 @@
+>>> db.query_to_map("select lntacode as nta, count(*) cnt from lion group by lntacode;", 'cnt')
+
+#will return a website with the map
+```
+
+
+![alt text](https://github.com/safety-analytics-mapping/ris/blob/docs/Capture.JPG?raw=true)
+
+[Back to Table of Contents](#pysqldb3-public-functions)
+<br>
+
+#### query_to_shp
+**`DbConnect.query_to_shp(self, query, path=None, shp_name=None, cmd=None, gdal_data_loc=GDAL_DATA_LOC,
+                     print_cmd=False, srid=2263)`**
+
+Generates shapefile output from the data returned from a query. 
+
+###### Parameters:
+ - **`query` str**: SQL query as string type; the query should ultimatley return data (ie. include a `select` statement with polygon geometry or must contain appropriate join attrbute) 
+ - **`shp_name` str, default None**: Output filename to be used for shapefile (should end in .shp)
+ - **`path` str, default None**: Folder path where the output shapefile will be written to, if none provided user input is required
+ - **`cmd` str, default None**: GDAL command to overwrite default behavior 
+ - **`gdal_data_loc` str, default None**: Path to gdal data, if not stored in system env correctly
+ - **`print_cmd` str, default None**: Option to print ogr command (without password)
+ - **`srid` int, default 2263**: SRID to manually set output to
+
+**Sample**
+
+```
+>>> from pysqldb3 import pysqldb3
+>>> db = pysqldb3.DbConnect(type='pg', server=server_address, database='ris', user='****', password='*******')
+>>> db.query_to_shp("select street, segmentid, geom from lion where street ='WATER STREET'", path=r'C:\Users\HShi\Documents', shp_name='sample_shp.shp', print_cmd=True)
+
+- Query failed: table "tmp_query_to_shp_hshi_2023_01_16_1544" does not exist
+
+
+- Query run 2023-01-16 15:44:01.916732
+        drop table tmp_query_to_shp_hshi_2023_01_16_1544
+ ogr2ogr --config GDAL_DATA "C:\Program Files (x86)\GDAL\gdal-data" -overwrite -f "ESRI Shapefile" "C:\Users\HShi\Documents\sample_shp.shp"  -a_srs "EPSG:2263" PG:"host=dotdevrhpgsql01 user=hshi dbname=ris password=*********" -sql "SELECT * FROM (select \"street\" , \"segmentid\" , \"geom\" from (select street, segmentid, geom from lion where street ='WATER STREET') q ) x"
+b''
+sample_shp.shp shapefile
+written to: C:\Users\HShi\Documents
+generated from: select \"street\" , \"segmentid\" , \"geom\" from (select street, segmentid, geom from lion where street ='WATER STREET') q
+
+```
+
+[Back to Table of Contents](#pysqldb3-public-functions)
+<br>
+
+#### table_to_shp
+**`DbConnect.table_to_shp(table, schema=None, strict=True, path=None, shp_name=None, cmd=None,
+                     gdal_data_loc=GDAL_DATA_LOC, print_cmd=False)`**
+
+Generates shapefile output from the data returned from a query. 
+
+###### Parameters:
+ - **`table` str**: SQL query as string type; the query should ultimatley return data (ie. include a `select` statement with polygon geometry or must contain appropriate join attrbute) 
```

Comment:
this looks like its from query_to_shp not table_to_shp

Reply:
ah ok, I saw in ris README this issue may also exists.

Reply:
ah yeah ok we should fix mistakes like this as we find them